pkg/operator/ceph/cluster/cluster.go

```diff
@@ -478,3 +477,37 @@ func (c *cluster) postMonStartupActions() error {
 
 	return nil
 }
+
+func (c *cluster) configureMsgr2() error {
+	// Set network encryption
+	monStore := config.GetMonStore(c.context, c.ClusterInfo)
+	encryptionSetting := "crc secure"
+	logger.Infof("setting msgr2 encryption mode to %q", encryptionSetting)
```

Comment:
This log info should move after the `if` below, in case `encryptionSetting` changes.